WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Details/OrderDetailsDataSourceTests.swift

```diff
@@ -431,11 +288,12 @@ final class OrderDetailsDataSourceTests: XCTestCase {
         XCTAssertNil(createShippingLabelRow)
     }
 
-    func test_create_shipping_label_button_is_not_visible_for_cash_on_delivery_order() throws {
+    func test_create_shipping_label_button_is_not_visible_when_order_is_eligible_for_card_present_payment() throws {
```

Comment:
We may want to explain that the shipping button should not be shown because the order is unpaid, not just because it's card payment eligible?

Reply:
As discussed, I wanted to be consistent with the business logic here, as we can have an unpaid order, but not card payment eligible. In that case we would show the shipping label button 😃